api/src/main/java/org/apache/any23/extractor/ExtractorGroup.java

```diff
@@ -78,6 +78,7 @@ public boolean allExtractorsSupportAllContentTypes() {
         return true;
     }
 
+    @SuppressWarnings("unlikely-arg-type")
```

Comment:
I've removed both the `SupressWarnings` javac annotation and thought about this one again.

I've decided to introduce `<omitVisitors>FindUnrelatedTypesInGenericContainer</omitVisitors>` which will omit spotbugs from identifying this issue as a bug. My justification is that, in this case, because the `ExtractorFactory#getSupportedMIMETypes()` supports wildcards, and as far as I can see there is currently no way to determine the full Collection (e.g. `*/*`) of supported `MimeType`'s then we cannot enforce another check whether an `ExtractorGroup` supports all `MimeType`'s. 

What do you think about this assessment?

Reply:
Sorry, I wasn't clear. The code inside this method has a bug, as _correctly_ identified by the spotbugs plugin. The code is testing if a `Collection<MIMEType>` contains a `String`. This will always return false, as a `String` _cannot_ be an instance of `MIMEType`.

Therefore, the method needs to be fixed, and the spotbugs plugin is doing its job correctly: no suppression of warnings is needed here.